log.go

```diff
@@ -0,0 +1,133 @@
+// Package log implements a featured and structured generic logger
+// interface designed to be used in core and third-party vinxi components.
+package log
+
+import (
+	"io"
+	"sync"
+	"time"
+
+	log "gopkg.in/Sirupsen/logrus.v0"
+)
+
+// mu provides thread synchronization for the public singleton API.
+var mu = sync.Mutex{}
+
+// Logger represents the standard default built-in logger (equivalent to default Logrus logger).
+var Logger = log.New()
+
+// New creates a new Logrus based logger.
+func New() *log.Logger {
+	return log.New()
+}
+
+// WithFields returns a new entry with `fields` set.
+func WithFields(fields log.Fields) *log.Entry {
+	return Logger.WithFields(fields)
+}
+
+// WithField returns a new entry with the `key` and `value` set.
+func WithField(key string, value interface{}) *log.Entry {
+	return Logger.WithField(key, value)
+}
+
+// WithError returns a new entry with the "error" set to `err`.
+func WithError(err error) *log.Entry {
+	return Logger.WithError(err)
+}
+
+// Debug level message.
+func Debug(msg string) {
+	Logger.Debug(msg)
+}
+
+// Info level message.
+func Info(msg string) {
+	Logger.Info(msg)
+}
+
+// Warn level message.
+func Warn(msg string) {
+	Logger.Warn(msg)
+}
+
+// Error level message.
+func Error(msg string) {
+	Logger.Error(msg)
+}
+
+// Fatal level message, followed by an exit.
+func Fatal(msg string) {
+	Logger.Fatal(msg)
+}
+
+// Debugf level formatted message.
+func Debugf(msg string, v ...interface{}) {
```

Comment:
It could be one, but that interface respects both Go method convention of `log` and the Logrus interface. I think it's better to keep it as it's basically a proxy/delegator to Logrus API.